src/SFA.DAS.RoATPService.Application.Api/Controllers/OrganisationController.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace SFA.DAS.RoATPService.Application.Api.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Threading.Tasks;
+    using Domain;
+    using MediatR;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Logging;
+    using Middleware;
+    using RoATPService.Api.Types.Models;
+    using Swashbuckle.AspNetCore.SwaggerGen;
+
+    [Authorize(Roles = "AssessorServiceInternalAPI")]
+    [Route("api/v1/[controller]")]
+    public class OrganisationController : Controller
+    {
+        private readonly ILogger<OrganisationController> _logger;
+        private readonly IMediator _mediator;
+
+        public OrganisationController(ILogger<OrganisationController> logger, IMediator mediator)
+        {
+            _logger = logger;
```

Comment:
_logger not used here?  Is that planned to do at a later date?

Reply:
Yes, will add logging when the validation checks are added in future stories

Reply:
Story ON-1436 has the details

Reply:
that's fine